packages/cli/src/index.ts

```diff
@@ -184,6 +184,10 @@ program
       command: 'mastra deploy',
       args,
       execution: async () => {
+        console.warn(`DEPRECATED: The deploy command is deprecated.
```

Comment:
I guess you want logger
```suggestion
        logger.warn(`DEPRECATED: The deploy command is deprecated.
```